### DISCUSSION TITLE
Keyboard layout changes, how do I disable that?

### DISCUSSION THREAD

**Discussion by ArgentKiEn** [AnySoftKeyboard/AnySoftKeyboard#3995] [Q&A]
If im typing in a text field I get a larger spacebar, when typing in a URL the spacebar skrinks and I get ".com" next to the spacebar. I get use to the position of the spacebar and the change throws me off. Ive tried searching for an option but I find the customization in the app a bit confusing. Any help would be great.

**Comment by shamiv** [comment]
I have exactly the same issue. Very annoying.

**Reply by ArgentKiEn** [reply]
I use OpenBoard now, much better imo.

**Comment by georg-d** [comment]
Did you try switching off 
DE: Benutzeroberfläche > Optimierungen und mehr > Aktiviere spezielle Eingabefeldmodi > URL-Eingabefeld 
(in EN probably User interface > optimizations and more > activate special input field modes > URL input  field)

**Comment by garrettec1** [comment]
https://github.com/AnySoftKeyboard/AnySoftKeyboard/discussions/4512
